src/view/screens/PostThread.tsx

```diff
@@ -79,9 +75,6 @@ export function PostThreadScreen({route}: Props) {
 
   return (
     <View style={s.hContentRegion}>
-      {isMobile && (
-        <ViewHeader title={_(msg({message: 'Post', context: 'description'}))} />
```

Comment:
This moved to always be in the flat list instead of sometimes-here-sometimes-there.